zng/docs/zson.md

```diff
@@ -808,19 +832,17 @@ the defines their type.
 
 <map> = "|{" <mlist> "}|"  |  "|{"  "}|"
```

Comment:
@mccanne: Out of curiosity, could maps be made more like records rather than lists, like this?
```
|{ <key>: <value>, <key>: <value>, ... }|
```

Reply:
That should work.  I'll give it a spin.

Reply:
This will involve changes to the spec, parser, and writer so I will merge this PR and put up a subsequent PR.  I think it looks better and is a good idea and I don't see why it won't work syntactically since it's parallel to a record and has differentiating grouping symbols.

Reply:
issue #1824